Rakefile

```diff
@@ -22,6 +22,9 @@ Hoe.spec 'net-http-persistent' do
 
   dependency 'connection_pool',   '~> 2.2'
   dependency 'minitest',          '~> 5.2', :development
+  dependency 'minitest',          '~> 5.2', :development
+  dependency 'hoe-travis',        '>= 1.3', :development
```

Comment:
hoe-travis must be version 1.4.1 or newer, and less than 2.x so the correct requirement is `'~> 1.4', '>= 1.4.1'`